rllib/algorithms/tests/test_callbacks.py

```diff
@@ -131,15 +141,17 @@ def test_before_sub_environment_reset(self):
             .callbacks(BeforeSubEnvironmentResetCallback)
         )
 
-        for _ in framework_iterator(config, frameworks=("tf", "torch")):
+        # Test with and without Connectors.
```

Comment:
Testing w/ and w/o connectors more important than different frameworks (env-runners are framework-agnostic).